sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/AppConfigurationRefresh.java

```diff
@@ -25,7 +30,9 @@ public interface AppConfigurationRefresh extends ApplicationEventPublisherAware
      * Checks configurations to see if configurations should be reloaded. If the refresh interval has passed and a
      * trigger has been updated configuration are reloaded.
      */
-    void refreshAsync();
+    default void refreshAsync() {
+        LOGGER.info("AppConfiguration.refreshAsync needs to be implimented to trigger a refresh.");
```

Comment:
Ideally, we should throw an `UnsupportedOperationException` here if this method is not implemented. Logging an info message will lead the user to believe that the refresh happened when it didn't.